package.json

```diff
@@ -66,6 +66,9 @@
 			"copyright": {
 				"description": "Displays copyright notice."
 			},
+			"delegate": {
+				"description": "Manages Lisk delegates."
```

Comment:
This is a bit misleading, and highlights that `Manages` is not really the right word for accounts or blocks. Well, maybe accounts. How about `Commands relating to Lisk delegates.`?